### ISSUE TITLE
Small Error in templatetags.seo

### ISSUE THREAD

**Issue by ghost** [willhardy/django-seo#17]
On line 59 you should insert 'template.' before TemplateSyntaxError(e.msg) and you should removed the e.msg part from the argument

```
if not metadata:
        # Fetch the metadata
        try:
            metadata = get_metadata(path, self.metadata_name, context, **kwargs)
        except Exception, e:
            raise TemplateSyntaxError(e.msg)
```

should be

```
if not metadata:
        # Fetch the metadata
        try:
            metadata = get_metadata(path, self.metadata_name, context, **kwargs)
        except Exception, e:
            raise template.TemplateSyntaxError(e)
```

based on the way you have done the imports at the top of this file :)

G

**Comment by willhardy** [comment]
Well found!

This also means that the exception is never properly raised in the tests...

**Comment by willhardy** [comment]
Actually it was: The `NameError` raised because of the bug you found was converted into a `TemplateSyntaxError`, which is what we wanted to raise anyway, and what the test was looking for :-)

**Comment by willhardy** [comment]
Closed by 702d8f00e28e0e7272819d72c582c3583dcc5b04. Raising a TemplateSyntaxError on our own terms.
